sdk/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [UPCOMING]
 
+### Added
+
+- Add `resolveEnsName` and `resolveEnsNameWithProvider` functions
```

Comment:
I'm not a huge fan of the naming conventions here. 

Resolve with provider doesn't require a provider.

Could this be better handled? 

Some options:

- `resolveENSNameWithNetwork`
- Create overloads for resolveENS, the type signature accepts a network OR provider, and then at runtime we switch based on what was detected.

Reply:
Seconded,

Reply:
This changelog is out of date as there is only 1 function: @josemarinas please resolve these once done.